pkg/notifications/notifiers_test.go

```diff
@@ -44,7 +43,7 @@ func TestNewNotifier(t *testing.T) {
 }
 
 func TestNewNotifiers(t *testing.T) {
-
+	// _, err := NewNotifiers("./testdata/invalid.toml")
```

Comment:
@gaurav-gogia could you take of these comments?

Reply:
sure ✌️